lib/main.js

```diff
@@ -189,6 +200,21 @@ export default {
     });
   },
 
+  async getProgram(Linter, configurationPath) {
+    let program;
+    const tsconfigPath = path.resolve(path.dirname(configurationPath), 'tsconfig.json');
```

Comment:
It looks like`Linter.findConfigurationPath` searches for a `tslint.json` (i.e. `CONFIG_FILENAME`) rather than a project’s `tsconfig.json`, which is what TSLint needs to create a program object: https://github.com/palantir/tslint/blob/617e705330a4e64f84d8d2759f684437203b75a8/src/linter.ts#L56-L59

Is there sample code demonstrating how to locate `tsconfig.json` using `Linter.findConfigurationPath`?